transmart-core-db/grails-app/services/org/transmartproject/db/user/UsersResourceService.groovy

```diff
@@ -66,4 +66,11 @@ class UsersResourceService implements UsersResource {
         } as List<CoreUser>
     }
 
+    @Transactional(readOnly = true)
+    @Override
+    List<CoreUser> getUsersWithEmailSpecified() {
+        User.withSession { session ->
```

Comment:
the left join produces one user row per role. Thus you would get list of users with duplicates. Remove the left join or deduplicate users afterward.

Reply:
thanks Ruslan, this is fixed now